{{cookiecutter.python_name}}/pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["jupyter_packaging~=0.7.0", "jupyterlab~=3.0.0b6", "setuptools>=40.8.0", "wheel"]
+requires = ["jupyter_packaging~=0.7.0", "jupyterlab>=3.0.0b6,==3.*", "setuptools>=40.8.0", "wheel"]
```

Comment:
it looks like right now that `jupyter_server` needs a pin:
```toml
[build-system]
requires = [
  "jupyter_packaging~=0.7.0",
  "jupyterlab_server ==2.0.0rc0",
  "jupyterlab==3.0.0b8",
  "setuptools>=40.8.0",
  "wheel",
]
build-backend = "setuptools.build_meta"
```

server `rc1` changed a function name which hasn't been released yet to match lab. Just wait for the next lab beta?

Reply:
Does the build system need this, or the runtime? Either way, this pin should normally be the responsibility of the package that imports it (`jupyterlab` / `jupyterlab_server`), which I think have been added?